core/src/main/resources/lib/form/entry.jelly

```diff
@@ -68,18 +68,18 @@ THE SOFTWARE.
   </j:if>
   <!-- expose this so that we can look up the @field value later from prepareDatabinding.jelly -->
   <j:set var="entry" value="${attrs}" />
-  <tr>
-    <td class="setting-leftspace"><st:nbsp/></td>
-    <td class="setting-name">
+  <div class='tr'>
+    <div class="setting-leftspace"><st:nbsp/></div>
+    <div class="setting-name">
       <j:out value="${escapeEntryTitleAndDescription ? h.escape(attrs.title) : attrs.title}" />
-    </td>
-    <td class="setting-main">
+      <f:helpLink url="${attrs.help}" featureName="${title}"/>
+    </div>
+    <div class="setting-main">
       <d:invokeBody />
-    </td>
-    <f:helpLink url="${attrs.help}" featureName="${title}"/>
-  </tr>
+    </div>
+  </div>
   <!-- used to display the form validation error -->
-  <tr class="validation-error-area"><td colspan="2" /><td /><td /></tr>
+  <div class="validation-error-area tr"><div colspan="2" /><div /><div /></div>
```

Comment:
closing div tags should be `</div>` instead of `<div />`

Reply:
actually, this `<div />` tag looks like it has been added by mistake, it does not do anything

Reply:
All of these are replacing TR/TD. I was lazy and not paying attention to the fact that those tags magically close. To properly emulate the existing don't, they should all have closing tags.

I'll update this later.